src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let robotCounter = 0;
+
+  return function(firstAction) {
```

Comment:
The parameter name 'firstAction' is not appropriate, consider renaming it to 'num1' or 'firstNumber'